### ISSUE TITLE
Can't share photos/videos

### ISSUE THREAD

**Issue by mjholtzem** [facebook/facebook-sdk-for-unity#163]
Since the "publish_actions" permission has been deprecated, screenshots can no longer be shared using the FB.API call. The SharePhoto, ShareVideo, and ShareMedia methods available in the Android and iOS SDKs need to be exposed to the Unity SDK

**Comment by MarkBurvs** [comment]
Same issue here. Just building a new app for a charity event this Summer, thought we'd just be able to do exactly what we've done several times before using publish_actions.

Would desperately love to hear any news on whether FBSDKSharePhoto is coming to the Unity plugin any time soon...?

**Comment by Drezus** [comment]
+1 👍: Currently, the only apparent way to get around this is to prepare your media extenally using your own server structure in order to generate a public hyperlink to that content, and share it via ShareLink. Of couse this is subpar to the full SharePhoto experience and we expect to see this supported very soon in Unity SDK.

Funny thing is: it didn't took that long at all to remove _publish_actions_ from use, but it does take months to consider even reimplementing functionality that the SDK already had to start with! What gives?

**Comment by JimmyDeemo** [comment]
I would be interested in finding out if this post; https://developers.facebook.com/blog/post/2018/07/31/platform-update-publish-permission/ is extended to Unity developers.

**Comment by JimmyDeemo** [comment]
No, in theory one could write a wrapper using the iOS and Android SDK, but we found it didn't play nice when we tried.

**Comment by arkarmintun1** [comment]
Image sharing to facebook from unity app still not available? Is there any workarounds for that?

**Comment by MixMasterRickyC** [comment]
We ended up using a native share to post images to Facebook.  https://github.com/yasirkula/UnityNativeShare
or from the asset store
https://assetstore.unity.com/packages/tools/integration/native-share-for-android-ios-112731
Hope this helps.

**Comment by smumair06** [comment]
Wow, still that part is untouched by facebook team.

**Comment by Mattias-TB** [comment]
I have submitted a bug report here: https://developers.facebook.com/support/bugs/1209896982758193/

If you have other methods that you know are missing, please add them to that report as well.

**Comment by Mattias-TB** [comment]
This should supposedly be fixed now according to the reply in the bug report. Has anyone tried it yet?
